src/Administration/Resources/app/administration/src/app/component/form/select/entity/sw-entity-single-select/sw-entity-single-select.html.twig

```diff
@@ -137,19 +137,18 @@
                     >
                         <!-- eslint-disable-next-line sw-deprecation-rules/no-twigjs-blocks -->
                         {% block sw_entity_multi_select_base_results_list_result_preview %}
-                        <template
-                            v-if="shouldShowActiveState"
-                            #preview
-                        >
-                            <!-- eslint-disable-next-line sw-deprecation-rules/no-twigjs-blocks -->
-                            {% block sw_entity_multi_select_base_results_list_result_active %}
-                            <sw-icon
-                                class="sw-entity-single-select__selection-active"
-                                size="6"
-                                :color="getActiveIconColor(item)"
-                                name="default-basic-shape-circle-filled"
-                            />
-                            {% endblock %}
+                        <template v-if="shouldShowActiveState">
+                            <slot name="preview">
+                                <!-- eslint-disable-next-line sw-deprecation-rules/no-twigjs-blocks -->
+                                {% block sw_entity_multi_select_base_results_list_result_active %}
```

Comment:
please don't add new twig blocks to the admin, refer here for the reasoning behind that:
https://www.shopware.com/en/news/new-strategies-for-administration-extensibility-in-shopware-6/

Reply:
It is not a new one, just one indentation level further down, as I needed to spilt the template element :⁠-⁠)

Reply:
ah sorry haven't seen that